Mathlib/Combinatorics/HalesJewett.lean

```diff
@@ -368,4 +506,31 @@ theorem exists_mono_homothetic_copy {M κ : Type*} [AddCommMonoid M] (S : Finset
     simp_rw [← hy, Option.map_some', Option.getD]
 #align combinatorics.exists_mono_homothetic_copy Combinatorics.exists_mono_homothetic_copy
 
+namespace Subspace
+
+/-- The **extended Hales-Jewett theorem**: For any finite types `η`, `α` and `κ`, there exists a
```

Comment:
```suggestion
/-- The **extended Hales-Jewett theorem**, also known as the **multidimensional Hales-Jewett theorem**: For any finite types `η`, `α` and `κ`, there exists a
```
"Multidimenensional" is more descriptive, and both seem to appear in the literature (see e.g. http://dx.doi.org/10.4007/annals.2012.175.3.6)

Reply:
You are right, I prefer "multidimensional" over "extended".